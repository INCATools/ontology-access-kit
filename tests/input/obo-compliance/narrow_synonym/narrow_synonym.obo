format-version: 1.4
ontology: narrow_synonym

[Term]
id: X:1
name: example term
synonym: "narrow synonym" NARROW []

