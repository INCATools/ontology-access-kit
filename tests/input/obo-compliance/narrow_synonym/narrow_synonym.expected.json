{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/narrow_synonym.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "lbl" : "example term",
      "type" : "CLASS",
      "meta" : {
        "synonyms" : [ {
          "pred" : "hasNarrowSynonym",
          "val" : "narrow synonym"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym",
      "lbl" : "has_narrow_synonym",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}